frontend/src/components/pages/CardDetails/index.jsx

```diff
@@ -146,6 +144,13 @@ function CardDetailsUnconnected({
     formErrors,
   };
 
+  if (
+    cards === undefined &&
```

Comment:
Why did you go with && instead of  || ?  
Why do you care about projectreviews and not topic reviews?

In any case, the current code has no unnecessary re-renders, so adding this extra logic doesn't seem to be something useful. Please remove it.